cax/cax.json

```diff
@@ -69,10 +71,15 @@
     "receive": "scp",
     "hostname": "tegner-login-1.pdc.kth.se",
     "directory": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_raw": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_root": "/cfs/klemming/projects/xenon/xenon1t/processed",
     "upload_options": [],
     "username": "bobau",
     "download_options": [
       "midway-login1"
+    ],
+    "pax_versions": [
+      "v4.10.0"
```

Comment:
but won't process.py see this "pax_version" and attempt to submit jobs? (at least that's how i implemented this option initially)